deploy/README.md

```diff
@@ -333,6 +327,16 @@ It is possible to migrate an existing Wikibase installation to WBS Deploy. The g
  - Regenerate the WDQS database
  - Regenerate the Elasticsearch database
 
+### Can I host WBS Deploy locally?
+
+Due to the OAuth configuration for MediaWiki and QuickStatements, along with the automatic SSL certification generated by Traefik, you must specify values for `WIKIBASE_PUBLIC_URL` and `QUICKSTATEMENTS_PUBLIC_URL` in your `.env` file. These values should be domain names that route to the IP address of the server hosting these services and must be accessible on the Internet.
+
+However, for testing purposes WBS Deploy can be run locally on a server that is not accessible to the Internet, with the following caveats:
+
+* In this configuration, you will still need to set `WIKIBASE_PUBLIC_URL` and `QUICKSTATEMENTS_PUBLIC_URL` to URLs that resolve locally to the IP address of the machine running the services. Configuring locally resolving DNS entries differs depending on your environment (Linux, MacOS, Windows), so setting this up correctly will require knowledge of or additional research about your specific setup. 
+* Any SSL certificates generated in this setup will be invalid, though you can optionally bypass the warning about these invalid certificates when first loading the Wikibase site in the browser.
+* QuickStatements will not function in this setup, as OAuth will not authorize against a local, non-Internet-accessible Wikibase installation.
```

Comment:
Why is this? Is the reason here that the certs on `WIKIBASE_PUBLIC_HOST` are invalid?

Reply:
A couple reasons: 1) yes--certs are invalid, and, 2) the container has to be able to resolve the DNS entry on the host network. I didn't explain these things in more detail here as I think that would be prone to confuse the user more than clarify. The intention here was to make clear what was possible, what we currently support and what we don't.

Reply:
I think we could explain that invalid certs are one of the reasons because we talk about them already.

Reply:
Really # 2 is the crux though. If you ran a local DNS which the Docker services knew about then theoretically things would be fine, except the certs wouldn't generate automatically through Let's Encrypt correctly as that path requires Internet DNS entries.

Reply:
Couldn't this be fixed with docker network hostname aliases?

Reply:
Re. docker names: I don't think so as they still won't be accessible by Let's Encrypt.

Again though the attempt here is to document the current state of things and what we know how to support now, as what was there was unclear and causing some confusion for at least a couple users. Not trying to fix or develop anything new.

Looking forward, if the truly local only scenario proves critical, we will need at least to provide guidance for generating local certs (or non-https) without relying on Let's Encrypt.